lib/utils/BvEvent.js

```diff
@@ -23,7 +23,9 @@ export class BvEvent {
         let defaultPrevented = false;
         // Recreate preventDefault method. One way setter.
         this.preventDefault = function preventDefault() {
-            defaultPrevented = true;
+            if (this.cancelable) {
```

Comment:
Yeah, true, 99% of the time this is used it would be for a `preventDefault()` use case.